### ISSUE TITLE
Why isn't it possible to keep a TabView during navigation?

### ISSUE THREAD

**Issue by NordlingArt** [NativeScript/NativeScript#4083]
As the question says. I do realize the logics behind it since a TabView lies under a specific page module. But I think a TabView is impractical since it disables the navigation ability of the app itself. Apps like Instagram still keep its tab view at the bottom despite navigating to another view.

I've been reading the discussions at #2094 and #1043. They're both old issues that were never taken care of. Why is that?

**tl;dr** a TabView should be able to stay in the application window and not inside a specific view controller.

**Comment by greenstevester** [comment]
+1 - I have tried hard to make tabs work correctly with NS Angular and just can't get navigation to work properly with Angular routes - my workaround so far is using modal from with a tab page.
I would also like to see this issue resolved, with a clear working example.

**Comment by NickIliev** [comment]
@NordlingArt , @greenstevester there is a discussion [here (also provided possible solutions at the end of the thread) ](https://github.com/NativeScript/NativeScript/issues/1043)and  [here ](https://github.com/NativeScript/NativeScript/issues/173)about very similar functionality so I will close this as a duplicate.

**Comment by vick16ach** [comment]
heyyy...

frameModule.topmost().navigate({
                        moduleName: "views/home/estimate/estimate",
                        backstackVisible: false,
                        animated: true,
                        transition: {
                            name: "slide",
                            duration: 500,
                            curve: "easeOut"
                        }
                    });

estimate.xml has a tabview in it.

how to navigate to tabview 1 in this case can you u guide me????
